spec/spec_helper_acceptance_local.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+configure_beaker do |host|
+  install_module_from_forge_on(host, 'puppet-archive', '6.1.1')
```

Comment:
if those are hard dependencies to the module, the should be added to the metadata.json instead